docs/draft-alakuijala-brotli-07.nroff

```diff
@@ -708,18 +713,18 @@ earlier. A complex prefix code must have at least two non-zero
 code lengths.
 
 The bit lengths of the prefix code over the code length alphabet
-are compressed with the following static prefix code (where the
-bits shown are reversed in the actual compressed stream):
+are compressed with the following static prefix code (as it appears
```

Comment:
Please replace 'static prefix code' with 'variable length code' here and a few paragraphs below. That is the terminology that we use for the other two such codes.

(and fix "from right to left")